pkg/nsx/services/firewall_test.go

```diff
@@ -63,7 +63,35 @@ var (
 		Spec: v1alpha1.SecurityPolicySpec{
 			AppliedTo: []v1alpha1.SecurityPolicyTarget{
 				{
-					PodSelector: &metav1.LabelSelector{MatchLabels: map[string]string{"pod_selector_1": "pod_value_1"}},
+					PodSelector: &metav1.LabelSelector{
+						MatchLabels: map[string]string{"pod_selector_1": "pod_value_1"},
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "k1",
+								Operator: metav1.LabelSelectorOpIn,
+								Values: []string{
+									"a1",
+									"a2",
+								},
```

Comment:
The indentation looks a little weird. It looks that you tried using multiple spaces to align the values, but not all of them actually align. And there are several such situations below.

Reply:
Thanks for bringing this attention. I've re-formatted the code to align each value filed.